url.bs

```diff
@@ -1626,8 +1623,9 @@ different document encoding. Using the <a>UTF-8</a> encoding everywhere solves t
 <h3 id=url-parsing>URL parsing</h3>
 
 <p>The <dfn export id=concept-url-parser lt="URL parser">URL parser</dfn> takes a string
-<var>input</var>, with an optional <a>base URL</a> <var>base</var> and an optional
-<a for=/>encoding</a> <var>encoding override</var>, and then runs these steps:
+<var>input</var>, with an optional null or <a>base URL</a> <var>base</var> (default null) and an
```

Comment:
Needs dfn (and since this is internal you can also update the call sites)

Reply:
See above.